src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,34 +1,102 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type AddMovieProps = {
+  onAdd: (movie: Movie) => void;
+};
+export const NewMovie: React.FC<AddMovieProps> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+
+  const [addTitle, setAddTitle] = useState('');
+  const [addDescription, setAddDescription] = useState('');
+  const [addImgUrl, setAddImgUrl] = useState('');
+  const [addImdbUrl, setaddImdbUrl] = useState('');
+  const [imdbId, setimdbId] = useState('');
```

Comment:
It's better to combine everything into formData useState
```suggestion
const [formData, setFormData] = useState({
    title: '',
    description: '',
    imgUrl: '',
    imdbUrl: '',
    imdbId: '',
  });
```